index.js

```diff
@@ -9,10 +9,61 @@ function ask(questionText) {
 
 start();
 
+let min = 1;
+let max = 100;
+let guess;
+let secretNumber;
+
 async function start() {
   console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
+  secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
+  guess = Math.floor((max - min) / 2);
+  let answer = await ask("Is this your number? Please answer (Y)es, (H)igher, (L)ower")
+  while (answer != 'Y'){
+    if (answer === 'Y') {
```

Comment:
you don't need this If statement here. If the user entered yes for an await ask, then the program will break out of the loop.